### ISSUE TITLE
[Table] Dialog里嵌套设置固定列的表格滚动时，固定列会挤压

### ISSUE THREAD

**Issue by ruiyong0416** [Tencent/tdesign-vue-next#3844]
### tdesign-vue-next 版本

1.8.0

### 重现链接

https://codesandbox.io/p/sandbox/base-dialog-forked-crzczm?file=%2Fsrc%2FDemo.vue%3A7%2C16

### 重现步骤

```
<template>
  <t-space>
    <t-button theme="primary" @click="onClick">打开弹窗</t-button>
    <t-dialog
      v-model:visible="visible"
      header="嵌套表格"
      width="300"
      destroy-on-close
    >
      <t-table bordered row-key="id" :columns="columns" :data="data" />
    </t-dialog>
  </t-space>
</template>
<script setup>
import { ref } from "vue";

const visible = ref(false);
const onClick = () => {
  visible.value = true;
};

const columns = [
  {
    title: "列1",
    colKey: "col_1",
    width: 100,
    fixed: "left",
  },
  {
    title: "列2",
    colKey: "col_2",
    width: 100,
    fixed: "left",
  },
  {
    title: "列3",
    colKey: "col_3",
    width: 100,
  },
];
const data = [
  {
    id: 1,
    col_1: "行1_1",
    col_2: "行1_2",
    col_3: "行1_3",
  },
  {
    id: 2,
    col_1: "行2_1",
    col_2: "行2_2",
    col_3: "行2_3",
  },
];
</script>

### 期望结果

滚动时固定列不挤压

### 实际结果

滚动时固定列被挤压

### 框架版本

3.3.9

### 浏览器版本

Chrome(120.0.6099.201)

### 系统版本

win11

### Node版本

16.16.0

### 补充说明

查看元素是表格的left计算不正确。
在监听弹窗打开后，延时100ms左右重新赋值表格数据是可以计算正确的。

**Comment by chaishi** [comment]
fixed https://github.com/Tencent/tdesign-vue-next/pull/3967

**Comment by uyarn** [comment]
fixed 1.9.0
